app.go

```diff
@@ -33,10 +33,12 @@ const (
 
 	serverSoftware = "WriteFreely"
 	softwareURL    = "https://writefreely.org"
-
-	softwareVer = "0.3"
 )
 
+// Software version can be set from git env using -ldflags
+var	softwareVer = "v0.3"
```

Comment:
Could you format this file with `gofmt`? And if we remove the **v** from `$(GITREV)` as mentioned before, we can remove it here.